### PR TITLE
added concurrentPrivateTxs script for more exciting demo

### PR DESCRIPTION
**Pull request by macfarla**
Not sure if we want this - it does add another npm dependency but adds an interesting scenario for demoing.
Adapted from web3js-eea https://github.com/ConsenSys/web3js-eea/tree/master/example/concurrentPrivateTransactions
Signed-off-by: Sally MacFarlane <sally.macfarlane@consensys.net>

### CODE REVIEW

File: files/besu/smart_contracts/package.json [hunk 1]
```diff
@@ -8,10 +8,11 @@
...
   "dependencies": {
+    "async-promise-pool": "^1.0.3",
+    "ethereumjs-tx": "1.3.7",
+    "fs-extra": "^10.0.0",
     "solc": "^0.7.0",
     "web3": "^1.2.11",
-    "web3-eea": "^0.9.0",
-    "ethereumjs-tx": "1.3.7",
-    "fs-extra": "^10.0.0"
+    "web3-eea": "^0.9.0"
```
Comment (pinges) [inline files/besu/smart_contracts/package.json:16]: should we use the new web3js-quorum here?

Comment (joshuafernandes) [reply files/besu/smart_contracts/package.json:16]: Is it cross compatible with besu or just quorum specific?

**Review by joshuafernandes — APPROVED** [review]
LGTM

File: files/besu/smart_contracts/scripts/concurrentPrivateTxs.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
+const Web3 = require("web3");
```
Comment (ricardolyn) [inline files/besu/smart_contracts/scripts/concurrentPrivateTxs.js:1]: we should add this file to the `package.json` as a specific task like `concurent-private-tx` so users can do `npm run concurent-private-tx`

**Pull request merged**